README.md

```diff
@@ -31,6 +31,10 @@ Allows you to interact with openEO backends from your own (local) Python environ
     
         pip install openeo
 
+* Installing with on-demand preview extra:
```

Comment:
I think these "extra" install variants should be documented at https://open-eo.github.io/openeo-python-client/installation.html  (where there is more room for in-depth discussion) instead of the README.md  (which should stay as minimal as possible)
(the README.md should properly link to that of course)

Reply:
I only added the basic explanation.